x-pack/plugin/esql/qa/testFixtures/src/main/resources/lookup-join.csv-spec

```diff
@@ -177,6 +197,31 @@ emp_no:integer | language_code:integer | language_name:keyword | country:text
 10005          | 5                     | null                  | Atlantis
 ;
 
+nonUniqueRightKeyOnTheCoordinatorCorrectOrdering
+// Same as above, but don't ignore the order completely. At least the emp_no col must remain correctly ordered.
```

Comment:
This one is new, because above we have to ignore the ordering and thus don't test that the left key ordering remains stable. This test does.